spec/system/provider_interface/provider_user_sets_up_organisation_permissions_spec.rb

```diff
@@ -153,7 +163,7 @@ def and_an_email_is_sent_to_managing_users_in_the_partner_organisations
 
     @another_ratifying_provider_users.each do |user|
       open_email(user.email_address)
-      expect(current_email.subject).to have_content t('provider_mailer.organisation_permissions_set_up.subject', provider: @training_provider.name)
+      expect(current_email.subject).to have_content t('provider_mailer.organisation_permissions_updated.subject', provider: @training_provider.name)
```

Comment:
Can we also test for at least one `set_up` email being sent?

Reply:
There is on [this line](https://github.com/DFE-Digital/apply-for-teacher-training/pull/5647/files/50efe4b1b3ba76ae89890f60c51dfbfd1b165a55..f46692e28f1d9976db364628029388adb1cf1a04#diff-f59a34ed8a03df2952586f887dd71519b5348e834db92208219f0a540cbee3f7R161)!